vendor/autoload.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// autoload.php @generated by Composer
+
+if (PHP_VERSION_ID < 50600) {
```

Comment:
Please add `"platform-check": false,` into the `composer.json`, then this condition will not generate.